### ISSUE TITLE
Add code and models

### ISSUE THREAD

**Issue by impredicative** [KimSoybean/ScratchDet#1]
This issue merely serves as a tracker. Please update it when the code and models are added for https://arxiv.org/abs/1810.08425

**Comment by zimenglan-sysu-512** [comment]
hi @KimSoybean 
do you still have the plan to release the code and model?
thanks.

**Comment by hust-kevin** [comment]
@KimSoybean in your paper, batchsize=128 means 128 imgs per gpu or total batch size on 4 gpu?

**Comment by KimSoybean** [comment]
@zimenglan-sysu-512 Hi, we will release the codes and models. But we will release them after the announcement of CVPR2019. I recommend you to reproduce the code on mmdetection, which is much faster than SSD-caffe. (Our method is so easy)

**Comment by KimSoybean** [comment]
@hust-kevin Hi, the accumulate batchsize is 128, in our code for Root-ResNet34, we put 16 images on each GPU (totally 4 GPUs), and the gradient is updated by the average of double iterations. You can see the source code in SSD-caffe.

**Comment by hust-kevin** [comment]
@KimSoybean thanks, I will try it on mmdetection, I have only 1 GPU. if I set 16 images per GPU,should I change lr to 0.05/8, what's more,  how to updated gradient by the average of double iterations in pytorch?

**Comment by KimSoybean** [comment]
@hust-kevin I think lr=0.03 is better, but it depends on your loss curves. If you have only one GPU, I recommend you to use GN instead of BN and train more epochs (I set totally 44 epochs on ssd of mmdetection ).

**Comment by hust-kevin** [comment]
thanks

**Comment by dby2017** [comment]
Do you still have the plan to release the code and model?
thanks.

**Comment by KimSoybean** [comment]
@dby2017 Recently we have trained the 512x512 models on mmdetection which took a long time. We will release codes and models when we finish it.

**Comment by liuchang8am** [comment]
Can the idea generalise to other base networks? E.g., resnet 18 layers. I tried the paper's idea by adding BN after each conv layer, and train from scratch with lr=0.01 and lr=0.05, however I'm getting worse results.

**Comment by pyupcgithub** [comment]
@KimSoybean where can i get the pretrained model VGG-16-BN ？

**Comment by liuchang8am** [comment]
any updates?

**Comment by E-Dreamer-LQ** [comment]
Do you still have the plan to release the code and model?
thanks.

**Comment by KimSoybean** [comment]
@Forgetmyfairy I have just finished my graduation thesis for undergraduates. This weekend I will upload them.
